plugins/operators/__init__.py

```
@@ -772,28 +749,37 @@ def _clear_frame_field_inputs(ctx, inputs):
 
-    field_keys = list(schema.keys())
-    field_selector = types.AutocompleteView()
-    for key in field_keys:
-        field_selector.add_choice(key, label=key)
+    field_names = ctx.params.get("field_names", None) or []
 
-    field_prop = inputs.enum(
-        "field_name",
-        field_selector.values(),
+    field_choices = types.AutocompleteView(allow_duplicates=False)
+    for key in schema.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)